scripts/deploy.ts

```diff
@@ -73,10 +73,13 @@ async function main() {
 		env.TLDRAW_ENV === 'staging' || env.TLDRAW_ENV === 'production' || env.TLDRAW_ENV === 'preview',
 		'TLDRAW_ENV must be staging or production or preview'
 	)
+	let currentStep = 1
+	// Previews have an extra step of aliasing the deployment and production builds also release the VS Code extension
+	const totalSteps = previewId || env.TLDRAW_ENV === 'production' ? 8 : 7
 
 	await discordMessage(`--- **${env.TLDRAW_ENV} deploy pre-flight** ---`)
 
-	await discordStep('[1/7] setting up deploy', async () => {
+	await discordStep(`[${currentStep++}/${totalSteps}] setting up deploy`, async () => {
```

Comment:
maybe worth pulling this logic into the `discordStep` function? i've generally tried to keep the top-level stuff in this function as basic as possible to make it clear how deploys work